{{cookiecutter.project_name}}/.mlops-setup-scripts/cicd/azure-devops.tf

```diff
@@ -160,10 +160,6 @@ resource "azuredevops_branch_policy_build_validation" "testing-ci-build-validati
       repository_ref = data.azuredevops_git_repository.repo.default_branch
       match_type     = "Exact"
     }
-
-    scope {
-      match_type = "DefaultBranch"
```

Comment:
Can you please add a comment as to why this is necessary/why it fixed the issue.

Reply:
I scoured the internet in search of any RESP API documentation for what the different ways to configure `scope` mean exactly. Unfortunately, I couldn't find any concrete documentation. The best I could find was here: https://registry.terraform.io/providers/microsoft/azuredevops/latest/docs/resources/branch_policy_build_validation#argument-reference

Here was my thinking process making this change: from the terraform doc, it seems like users can configure multiple scopes. My assumption is that these scopes are clauses for where to configure this branch policy. The existing `Exact` scope seems to already accomplish what we want, which is matching to the default branch of the user specified repo. Then, with regards to the `DefaultBranch` scope, the terraform doc specified that users cannot configure `repository_id` nor `repositor_ref` when the `DefaultBranch` match type is chosen. My interpretation here is that that means that the `DefaultBranch` match type configures the policy on a project level, which is coherent my dogfooding experience of the old code.

Hence, I feel pretty confident that removing this scope is the fix that we need.